addons/project/models/project.py

```diff
@@ -632,6 +632,9 @@ def name_create(self, name):
     def create(self, vals_list):
         # Prevent double project creation
         self = self.with_context(mail_create_nosubscribe=True)
+        for vals in vals_list:
+            if not vals.get('label_tasks'):
```

Comment:
```suggestion
            if 'label_tasks' in vals and not vals['label_tasks']:
```
or else, if it is not in vals, it will be overridden.